spec/spec_helper.rb

```diff
@@ -8,6 +8,8 @@ def horizontal_rule(width = 5)
 
 require 'puppetlabs_spec_helper/module_spec_helper'
 
+FUTURE_PARSER = ENV['FUTURE_PARSER'] == 'true' || ENV['FUTURE_PARSER'] == 'yes'
+
```

Comment:
FUTURE PARSER testing has been disabled until we make a decision on which way to go.